mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_foreach_slices.mlir

```diff
@@ -171,41 +171,44 @@ module {
     // The same slice, but with dynamic encoding.
     // TODO: Investigates why reusing the same %tmp above would cause bufferization
     // errors.
-    %tmp1 = sparse_tensor.convert %sa : tensor<8x8xf64> to tensor<8x8xf64, #CSR>
-    %a_dyn = tensor.extract_slice %tmp1[%c1, %c1][%c4, %c4][%c1, %c2] : tensor<8x8xf64, #CSR> to
-                                                                        tensor<?x?xf64, #CSR_SLICE_DYN>
+    //
+    // FIXME: The canonicalizer for tensor.extract_slice does not work with sparse tensors.
```

Comment:
This might need more work... Maybe we should have our own `extract_slice` operation?